### ISSUE TITLE
Make Chaturbate and its followed up cams on one page like Cam4

### ISSUE THREAD

**Issue by Meetrying** [dobbelina/repository.dobbelina#1327]
### Prerequisites

- [X] I have searched the issue tracker to avoid opening a duplicate issue.

### Description

Similar to Cam4, it will make moving back and forth between pages much easier

**Comment by frankee123** [comment]
I don't want to intrude on this post, but felt like it's easier to comment here instead of making another separate thread about a chaturbate issue. It seems that some girls that are online are simply showing up as blank or if they are not online in my favorite folders. Some of them have the same still picture as if they are online from previous day which they are not. It seems the only way to update the pictures would be to hit refresh images which I do not want to do everytime to update the pictures. There is also an issue which some girls that are online and should be showing up if I click on the chaturbate webcams online favorites folders are simply not showing up on there. I know there are quite a few issues here, but I hope somebody can figure out how to fix them please. Thank you so much.

**Comment by 12asdfg12** [comment]
@frankee123 
have you tried the "Auto refresh webcam images" option in the settings?

**Comment by frankee123** [comment]
@12asdfg12 Yes sir, I've always had that enabled. The only thing that works is clicking on my chaturbate online favorites folder, but some girls that are online simply do not show up in it for some reason. My main Favorite videos folder only shows a few current images of girls that are online, and that is actually correct. It still shows me still pics from days prior on some girls, as well as some girls that I know are online, it doesn't show any image at all. I never had to click on the Refresh images button everytime for images to work correctly when I click on my favorites folder, but something isn't showing them correctly for some reason. For instance, one girl in particular wil lnot show up in my Favorites fodler as well as my Chaturbate online folder. She only shows up in one the the female tag folders for Chaturbate which is "fit." I don't understand it. Ty for the help.

**Comment by 12asdfg12** [comment]
Cumination uses a Chaturbate API that returns all rooms online, but it doesn't work 100% correctly, some rooms don't appear in the list even though they are online.
Try using "Followed Cams" which works correctly.

**Comment by frankee123** [comment]
@12asdfg12 thx bro, but I think I need cloudflare to make that work, and it doesn't on my android box sadly.

**Comment by dvgm506** [comment]
Models have an option that results in them not being in the online rooms list.

Maybe buy a raspberry pi and install flaresolverr? There is a windows version IIRC.

Bonus with the pi would be to use pihole for whole network ad blocking (beware, default lists might have some of the sites you use cumination for).

**Comment by frankee123** [comment]
Thx for the response. I wasted a lot fo money buying random new android boxes that all say 4k and have a lot of storage etc. I find myself using my xbox series x for kodi lately since it goes lightning fast and can play all 4k vids no problem lol. I just want to buy a cheap android box only for kodi and that can play 4k videos properly. I have had terrible luck with electronics my entire life I swear.
